test/cql-pytest/test_ssl.py

```diff
@@ -62,7 +62,13 @@ def try_connect(orig_cluster, ssl_version):
         # slow debug build running on a very busy machine, they may not be.
         # so let's increase them to 60 seconds. See issue #11289.
         connect_timeout = 60,
-        control_connection_timeout = 60)
+        control_connection_timeout = 60,
+        # The default reconnection policy has a large maximum interval
+        # between retries (600 seconds). In tests that restart/replace nodes,
+        # where a node can be unavailable for an extended period of time,
+        # this can cause the reconnection retry interval to get very large,
+        # longer than a test timeout.
+        reconnection_policy = cassandra.policies.ExponentialReconnectionPolicy(1.0, 4.0))
```

Comment:
Again, this isn't relevant to this specific test, which never needs to reconnect.